packages/providers/fal/image-to-video/fal_image_to_video/models/base.py

```
@@ -0,0 +1,212 @@
+        handler = fal_client.submit(self.endpoint, arguments=arguments)
+        request_id = handler.request_id
+        print(f"📤 Request submitted: {request_id}")
+
+        while True:
+            status = fal_client.status(self.endpoint, request_id, with_logs=True)
+            print(f"   Status: {status.status}")
+
+            if status.status == "COMPLETED":
+                return fal_client.result(self.endpoint, request_id)
+            elif status.status == "FAILED":
+                raise Exception(f"Generation failed: {status}")
```

Ruff (0.14.10)

Create your own exception

(TRY002)